examples/type_inference/type_inference_visitor_unittest.py

```diff
@@ -0,0 +1,243 @@
+"""Unittest for the type_inference_visitor."""
+
+import unittest
+import sys
+import astroid
+from typing import Tuple, List, Dict
+from astroid import test_utils
+from type_inference_visitor import TypeVisitor, set_const, set_binop, \
+    set_unaryop, set_dict, set_list, set_tuple
+
+
+class SetConstFunctionTest(unittest.TestCase):
+    """testers specifically for the function set_const using single nodes."""
+
+    def test_const_str(self):
+        """testing if the function set_const has the correct output for node
+        type str."""
+        expected = str
+        node = test_utils.extract_node("""a = __('sample_string')""")
```

Comment:
I tried without an assignment, this is the error I got: 
`Error
Traceback (most recent call last):
  File "/Users/HL/Desktop/pyta/tests/type_inference/type_inference_visitor_unittest.py", line 19, in test_const_str
    node = test_utils.extract_node("""'sample_string'""")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/astroid/test_utils.py", line 163, in extract_node
    extracted.append(tree.body[-1])
IndexError: list index out of range`